src/main/java/nl/thehyve/ocdu/validators/patientDataChecks/StudyPatientDataCheck.java

```diff
@@ -0,0 +1,28 @@
+package nl.thehyve.ocdu.validators.patientDataChecks;
+
+import nl.thehyve.ocdu.models.OCEntities.Subject;
+import nl.thehyve.ocdu.models.OcDefinitions.MetaData;
+import nl.thehyve.ocdu.models.errors.ValidationErrorMessage;
+import org.apache.commons.lang3.StringUtils;
+
+/**
+ * Created by bo on 6/15/16.
+ */
+public class StudyPatientDataCheck implements PatientDataCheck{
+
+    @Override
+    public ValidationErrorMessage getCorrespondingError(int index, Subject subject, MetaData metaData) {
+
+        String ssid = subject.getSsid();
+        String commonMessage = getCommonErrorMessage(index, ssid);
+
+        ValidationErrorMessage error = null;
+        String study = subject.getStudy();
+
+        if(StringUtils.isBlank(study)) {
+            error = new ValidationErrorMessage(commonMessage + "Study should be provided.");
+        }
+
+        return error;
```

Comment:
This check on the other hand does not look like a draft but it returns error for each data point. Was it intended? Do you check number of offenders in the Checks Runner class to filter them out before returning?